bleep-cli/src/scala/bleep/internal/Version.scala

```diff
@@ -0,0 +1,238 @@
+/*
```

Comment:
I guess this file is copy/pasted from scala steward? are there any changes? on other words: is there any need to review anything in here?

Reply:
The snippet in the top comment is added to it. And I removed this
implicit val versionCodec: Codec[Version] = deriveUnwrappedCodec
As it uses circe-generic-extras and I don't see any reason we need the codec.